be/src/exec/es/es_predicate.cpp

```diff
@@ -178,7 +179,7 @@ EsPredicate::EsPredicate(ExprContext* context,
 
 EsPredicate::~EsPredicate() {
     for(int i=0; i < _disjuncts.size(); i++) {
-        delete _disjuncts[i];
+        SAFE_DELETE(_disjuncts[i]);
```

Comment:
It is safe to `delete nullptr`, and `_disjuncts` is cleared later on, so keep using delete is OK